### PR TITLE
Nav secondary docs

### PR DESCRIPTION
**Pull request by APJohns**

### CODE REVIEW

File: src/app/modules/component-library/library/nav-secondary/api/api.component.html [hunk 1]
```diff
@@ -1,7 +1,114 @@
...
+              <p>When "true" the nav is collapsed.</p>
+            </td>
+          </tr>
+          <tr>
+            <td>areItemsExpanded</td>
+            <td>boolean</td>
+            <td>Determines if the nested navigation items should be expanded.</td>
+            <td>
+              <p>Default is "false".</p>
+              <p>When "true" the nested navigation items are all expanded.</p>
```
Comment (kaseyllabelle) [inline src/app/modules/component-library/library/nav-secondary/api/api.component.html]: "are all expanded" reads like all of the collapsibles are expanded. Is that true, or is it just one collapsible?

File: src/app/modules/component-library/library/nav-secondary/api/api.component.html [hunk 2]
```diff
@@ -1,7 +1,114 @@
...
+            </td>
+          </tr>
+        </tbody>
+      </table>
+
+      <h2 id="navItemDocs">NavItem</h2>
+      <p>There are two kinds of NavItems. The other is a parent node whose children are routing nodes that represents a grouping of NavItems.</p>
```
Comment (kaseyllabelle) [inline src/app/modules/component-library/library/nav-secondary/api/api.component.html]: I think we should add the explanation of both here if we're starting out saying there are 2 kinds.

Comment (kaseyllabelle) [reply src/app/modules/component-library/library/nav-secondary/api/api.component.html]: Does the parent node represent the grouping or do the children represent the grouping?

If parent, add commas for clarity.
"The other is a parent node, whose children are routing nodes, that represents a grouping of NavItems."

If children, remove s from represents.
"The other is a parent node whose children are routing nodes that represent a grouping of NavItems."

File: src/app/modules/component-library/library/nav-secondary/api/api.component.html [hunk 3]
```diff
@@ -1,7 +1,114 @@
...
+      <h3>Routing Node</h3>
+      <p>The routing node represents a link to another page in the application. This node is defined by providing the following properties to a NavItem object...</p>
```
Comment (kaseyllabelle) [inline src/app/modules/component-library/library/nav-secondary/api/api.component.html]: We can swap ellipses for period here.

File: src/app/modules/component-library/library/nav-secondary/api/api.component.html [hunk 4]
```diff
@@ -1,7 +1,114 @@
...
+            <th class="pt-0">Property</th>
+            <th class="pt-0">Type</th>
+            <th class="pt-0">Description</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>route</td>
+            <td>string</td>
+            <td>The route used for this NavItem.</td>
```
Comment (kaseyllabelle) [inline src/app/modules/component-library/library/nav-secondary/api/api.component.html]: We say "this NavItem" here and "the NavItem" on line 81. Can we choose one?

File: src/app/modules/component-library/library/nav-secondary/api/api.component.html [hunk 5]
```diff
@@ -1,7 +1,114 @@
...
+          </tr>
+          <tr>
+            <td>text</td>
+            <td>string</td>
+            <td>The text to use for the NavItem.</td>
+          </tr>
+        </tbody>
+      </table>
+      <h3>Parent Node</h3>
+      <p>The parent node has children that are routing nodes, and represents a grouping of NavITems. This node is defined by providing the following properties to a NavItem object...</p>
```
Comment (kaseyllabelle) [inline src/app/modules/component-library/library/nav-secondary/api/api.component.html]: Can we make this read more closely to the intro paragraph? We can also swap ellipses for period here. Update casing for 't' in NavItems.

File: src/app/modules/component-library/library/nav-secondary/api/api.component.html [hunk 6]
```diff
@@ -1,7 +1,114 @@
...
+      <table class="table table-bordered-rows mb-5">
+        <caption class="sr-only">Landing page header inputs</caption>
```
Comment (kaseyllabelle) [inline src/app/modules/component-library/library/nav-secondary/api/api.component.html]: Update caption

File: src/app/modules/component-library/library/nav-secondary/api/api.component.html [hunk 7]
```diff
@@ -1,7 +1,114 @@
...
+            <th class="pt-0">Property</th>
+            <th class="pt-0">Type</th>
+            <th class="pt-0">Description</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>children</td>
+            <td>NavItem[]</td>
+            <td>An array of NavITems that will be nested under a collapsible.</td>
```
Comment (kaseyllabelle) [inline src/app/modules/component-library/library/nav-secondary/api/api.component.html]: Update casing for 't' in NavItems.
We can make the voice more active here: "An array of NavItems nested in a collapsible."

File: src/app/modules/component-library/library/nav-secondary/api/api.component.html [hunk 8]
```diff
@@ -1,7 +1,114 @@
...
+          </tr>
+          <tr>
+            <td>text</td>
+            <td>string</td>
+            <td>The text to use for the xollapsible NavItem.</td>
```
Comment (kaseyllabelle) [inline src/app/modules/component-library/library/nav-secondary/api/api.component.html]: Swap x to c in collapsible

File: src/app/modules/component-library/library/nav-secondary/api/api.component.html (hunk 1 above)
Comment (APJohns) [reply src/app/modules/component-library/library/nav-secondary/api/api.component.html]: They are all expanded. Its an input on the component that when true expands all of them.

**Pull request merged**